src/less/projects.less

```diff
@@ -0,0 +1,125 @@
+.projects-grid-container {
+	background: white;
+}
+
+.project-container {
+	background: white;
+	height: 200px;
+	margin: 3px;
+	padding: 0;
+}
+
+.panel-project {padding: 0;}
+
+.panel-project > .panel-heading {color: white;}
+
+.smallest {
+	font-size: 10px;
+	letter-spacing: 1.3px;
+}
+
+pre.projects {
+	background-color: transparent;
+	border: none;
+	font-size: small;
+}
+
+
+
+.table-hover>tbody>tr:hover>td, .table-hover>tbody>tr:hover>th {
```

Comment:
Can this be scoped? - in fact, can this whole less section be scoped by a parent selector?